src/common/general/iconPathUtil.ts

```diff
@@ -0,0 +1,42 @@
+import * as vscode from "vscode";
+import * as path from "path";
+import ContextProvider from "../vscode/extension/ContextProvider";
+
+const DEFAULT_LIGHT_RESOURCES_FOLDER_PATH = "resources/light";
+const DEFAULT_DARK_RESOURCES_FOLDER_PATH = "resources/dark";
```

Comment:
Are there any possibility that theme resources could have another folder path? If not so, naming here could be misleading later on.

Reply:
currently not, shall I remove the "DEFAULT_" part for now?

Reply:
Hmm, could be better but it's up to you and not that essential 😄 I just wanted to point that.

Reply:
I will remove it 👍